BotTest/EmailVerificationTest.cs

```diff
@@ -0,0 +1,87 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using TU20Bot;
+using TU20Bot.Commands;
+using TU20Bot.Configuration;
+using Discord;
+using System.Linq;
+
+namespace BotTest {
+    [TestClass]
+    public class EmailVerificationTest {
+
+        private static EmailVerification _emailVerification;
+        private static EmailChecker _emailChecker;
+        private static Config _config;
+        private static Client _client;
+        private static Handler _handler;
+
+        [ClassInitialize]
+        public static async Task ClassInitialize(TestContext testContext) {
+
+            string token = "";
+
+            // Start bot with token from "token.txt" in working folder.
+            try {
+                var _token = File.ReadAllText("token.txt").Trim();
+                token = _token.ToString();
```

Comment:
`_token` is a `String` already.

```suggestion
                token = File.ReadAllText("token.txt");
```